app/dashboards/hold_dashboard.rb

```diff
@@ -17,8 +17,8 @@ class HoldDashboard < Administrate::BaseDashboard
     case_number: Field::String,
     status: Field::Select.with_options(searchable: false, collection: ->(field) { field.resource.class.send(field.attribute.to_s.pluralize).keys }),
     processing_notes: Field::Text,
-    created_at: Field::DateTime,
-    updated_at: Field::DateTime,
+    created_at: Field::DateTime.with_options(format: "%a, %d %b %Y"),
```

Comment:
I'm curious if we should match the display we have for the `DATE REQUESTED` etc which is `2021-02-18` instead of `Thu, 18 Feb 2021` for consistency within the same UI. Totally not blocking, but I sort of predict we'll eventually get that feedback so maybe we should just do it? I'm fine with merging as-is or normalizing in the same UI as there was no explicit stakeholder guidance I think

Reply:
Yeah, that makes sense. I asked Stephanie about it yesterday, and since neither of us had a preference, I figured I would just remove the timestamp and leave any specific formatting requests up to Mikki. However, it's cleaner and easier to just make it `Field::Date`, so I think we can roll with that approach for now.